lib/ecto_enum.ex

```diff
@@ -57,18 +64,34 @@ defmodule EctoEnum do
     end
   end
 
+  defmodule UndeterminableStorageError do
+    defexception [:message]
+
+    def exception(kw) do
+      msg = "cannot determine storage type for #{inspect kw}"
```

Comment:
Can we change this error msg to display something like `"Data type is invalid. EctoEnum only stores integers or strings."`?